src/aat/resources/features/F-1005 - Start Case Event TTL/F-1005.feature

```diff
@@ -0,0 +1,360 @@
+@F-1005
```

Comment:
Needs renaming (including the accompanying scenario and other TD files) to a unique feature number. as GlobalSearch has taken F-1005.